kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkTask.java

```diff
@@ -183,6 +184,17 @@ public void put(Collection<SinkRecord> records) throws ConnectException {
       String topic = record.topic();
       int partition = record.kafkaPartition();
       TopicPartition tp = new TopicPartition(topic, partition);
+      if (record.value() == null) {
+        if (connectorConfig.nullValueBehavior()
+            .equalsIgnoreCase(BehaviorOnNullValues.IGNORE.toString())) {
+          log.debug("Null valued record cannot be written to output as Avro. "
+              + "Skipping. Record Key: {}", record.key());
```

Comment:
probably print the topic, partition or offset. sending the key into logs may be unsafe.